.github/workflows/ci.yml

```diff
@@ -44,6 +44,63 @@ jobs:
     - run: npm run lint
     - run: npm run flow
     - run: npm test -- --maxWorkers=2 --ci
+    - name: Build Linux Packages
+      if: runner.os == 'Linux'
+      run: |
+        set -x
+        node scripts/release --linux
+    - name: Persist rpm package
+      if: runner.os == 'Linux'
+      uses: actions/upload-artifact@v1
+      with:
+        name: linux-release
+        path: ./dist/installers
+
+  integration_test_centos:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        node-version: [12.x]
+    container:
+      image: centos:8
+      env:
+        # This environment variable causes chrome to run with no sandbox- needed
+        # whenever chrome is run as root.
+        APPVEYOR: true
+    needs: build
+    steps:
+    - run: dnf install -y xorg-x11-server-Xvfb git nss-tools pulseaudio
+    - run: git config --global core.autocrlf false
+    - uses: actions/checkout@v2
+    - name: setup node version ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Set NPM Cache Directory
+      id: set-npm-cache-dir
+      run: echo "::set-output name=npm_cache_dir::$(npm config get cache)"
+    - name: Clear Extraneous Runner Cache
+      run: rm -rf "${NPM_CACHE:?}"
+      env:
+        NPM_CACHE: ${{ steps.set-npm-cache-dir.outputs.npm_cache_dir }}
+    - name: Cache node modules
+      uses: actions/cache@v1
+      env:
+        cache-name: cache-node-modules-ci-v4
+      with:
+        path: ${{ steps.set-npm-cache-dir.outputs.npm_cache_dir }}
+        key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ matrix.node-version }}-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-build-${{ env.cache-name }}-${{ matrix.node-version }}-
+    - run: npm install --no-audit
+    - name: Download rpm package
+      uses: actions/download-artifact@v1
+      with:
+        name: linux-release
+    - name: Integration Tests (Centos 8)
+      run: |
+        dnf install -y ./linux-release/brim_x86_64.rpm
+        xvfb-run -d -s "-screen 0 1280x1024x24" npm run itest -- --ci --forceExit
```

Comment:
What does `xvfb-run -d` do?

Reply:
The newer version of `xvfb` which centos8 has but ubuntu 18 does not says that `--auto-servernum` is deprecated and to use `-d` instead.

Reply:
TIL. Thanks!